src/gluonts/nursery/sagemaker_sdk/utils.py

```diff
@@ -19,7 +19,7 @@
 def make_metrics(metrics_names):
     avg_epoch_loss_metric = {
         "Name": "training_loss",
-        "Regex": r"'avg_epoch_loss'=(\S+)",
+        "Regex": r"'epoch_loss'=(\S+)",
```

Comment:
There is no avg_epoch_loss?

Reply:
If you look into the logs, there is one, but not in lines with "INFO:...", so apparently the metric extraction only works on info lines, thats why its not showing up in the charts.

Reply:
I verified this: indeed, `avg_epoch_loss` is displayed in the lines produced by tqdm, not in the logging lines